src/index.js

```diff
@@ -4,42 +4,166 @@ import _ from 'lodash';
 
 import './style.css';
 
-const tasks = [
-  {
-    description: 'Reapir the car',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Copmplete the Projects',
-    completed: false,
-    index: 2,
-  },
-  {
-    description: 'Submit the project',
-    completed: false,
-    index: 3,
-  },
-];
+class TaskList {
+  allTasks = [];
+  // tasks = JSON.stringify(this.allTasks);
+
```

Comment:
- [OPTIONAL] I think it would be better to remove unused code from your project rather than comment it out to keep your code clean and follow [the best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)💯

- [OPITONAL] Kindly remove the first three lines in this file too, I think you don't need to import `loadash` 👍
```js
/* eslint-disable */
import _ from 'lodash';
/* eslint-enable */
```

![image](https://user-images.githubusercontent.com/45231276/181750281-0d99a210-25c7-4b54-b455-141811a9c361.png)